### ISSUE TITLE
Support for UID and GID

### ISSUE THREAD

**Issue by rondegroot** [rb-dock8s/serviio#16]
Hello,

Is it possible to use a UID and GID to run Serviio in? Within the linuxserver.io containers you can use te PGID and PUID environment variables.

Regards, Ron

**Comment by ergoz** [comment]
@rondegroot yes, it is in todo. When i will find more time for this project i will make a new release
